src/Common/gui/widgets/MapWidget.cpp

```diff
@@ -394,7 +394,7 @@ void MapWidget::drawCountriesLegend(QPainter &p)
     auto legendHeight = (flagSize.height() + TEXT_MARGIM) * uniqueCountryMarkers.size() + TEXT_MARGIM;
 
     QSizeF legendSize(legendWidth, legendHeight);
-    QPointF legendTopLeft(width() - legendSize.width() - TEXT_MARGIM, TEXT_MARGIM);
+    QPointF legendTopLeft((width() - legendSize.width() - TEXT_MARGIM) / 2, TEXT_MARGIM);
```

Comment:
Nice @jonjamcam !

Maybe is better center vertically too. 

Try replace the second parameter (TEXT_MARGIM) by
```
height()/2 - legendSize.height()/2
```

Reply:
keeping above syntax too for horizontal parameter:

![image](https://user-images.githubusercontent.com/15310433/53452006-f405ad80-39fe-11e9-9cfd-b962bcafd12f.png)